examples/TestNetCoreConsole/Program.cs

```diff
@@ -123,7 +129,14 @@ static async Task Main(string[] args)
                 Console.WriteLine("Press a key to stop recording...");
                 Console.ReadKey(true);
 
-                signaler.Stop();
+                Console.WriteLine("Removing data channels...");
+                foreach (var dataChannel in pc.DataChannels.ToImmutableArray())
+                {
+                    Console.WriteLine($"Removing DataChannel {dataChannel.Label}: State '{dataChannel.State}'");
+                    pc.RemoveDataChannel(dataChannel);
+                }
+
+                await Task.Delay(TimeSpan.FromSeconds(10));
```

Comment:
Why the wait here?

Reply:
Seems like a big leftover actually. I believe it was added to give me some time to check that the data channels were removed before disposing the entire peer connection, but I'm not sure! I'll remove the wait

Reply:
made a fixup, just let me know and i'll rebase the commits so the fixup does not go on main
d328c3c5ce5d7ce04e3fe6e5ad670bfddafc52ad

Reply:
I'll squash anyway, by policy. We rarely merge, unless there's a very compelling reason to keep the history.